zerver/lib/webhooks/git.py

```diff
@@ -23,6 +23,8 @@
 PULL_REQUEST_OR_ISSUE_ASSIGNEE_INFO_TEMPLATE = u"(assigned to {assignee})"
 PULL_REQUEST_BRANCH_INFO_TEMPLATE = u"\nfrom `{target}` to `{base}`"
 
+PING_MESSAGE_TEMPLATE = u"{user_name} set up a webhook"
```

Comment:
This should probably clarify which integration it was?  E.g. "{user_name} successfully configured the {integration_name} webhook!" and pass in e.g. "GitHub".